apps/src/aichat/views/assets/UploadButton.tsx

```diff
@@ -108,10 +136,21 @@ const UploadButton: React.FC<{isDisabled: boolean}> = ({isDisabled}) => {
     true
   );
 
+  const onDeviceUploadClick = () => {
+    openFileInput();
+    dispatch(
+      sendAnalytics(EVENTS.AICHAT_MULTIMODAL_UPLOAD_OPENED, {
+        source: AssetSource.PROJECT,
+      })
+    );
+  };
+
   if (!currentChannelId) {
     return null;
   }
 
+  // need to figure out how to have onclick for analytics event,
+  // as well as default onclick that opens/closes dropdown
```

Comment:
Is this comment still relevant? Looks like we're doing both for each onclick right?